LibsAndSamples.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.1.32210.238
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.32428.217

```

Comment:
I wonder why this would be a downgrade?
Is this needed?

Reply:
I use VS2019. It's autogenerated and don't think this affects anything.